server/src/main/java/org/opensearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -1189,10 +1189,10 @@ List<String> getIndexSettingsValidationErrors(
             validationErrors.addAll(validatePrivateSettingsNotExplicitlySet(settings, indexScopedSettings));
         }
         if (indexName.isEmpty() || indexName.get().charAt(0) != '.') {
-            // Apply aware replica balance only to non system indices
+            // Apply aware replica balance validation only to non system indices
             int replicaCount = settings.getAsInt(
                 IndexMetadata.SETTING_NUMBER_OF_REPLICAS,
-                INDEX_NUMBER_OF_REPLICAS_SETTING.getDefault(Settings.EMPTY)
+                DEFAULT_REPLICA_COUNT_SETTING.get(this.clusterService.state().metadata().settings())
```

Comment:
We need to ensure system indices do not break

Reply:
This validation will not be applied on the system index
>        if (indexName.isEmpty() || indexName.get().charAt(0) != '.') { <-- line 1191